packages/block-library/src/group/variations.js

```diff
@@ -4,12 +4,21 @@
 import { __ } from '@wordpress/i18n';
 
 const variations = [
+	{
+		name: 'group',
+		title: __( 'Group' ),
+		description: __( 'Blocks shown in a column.' ),
+		attributes: { layout: { type: 'default' } },
+		scope: [ 'transform' ],
```

Comment:
Should we also add here the `inserter` scope and set it as default (`isDefault:true`)?